packages/fxa-metrics-processor/package.json

```diff
@@ -39,8 +39,9 @@
     "typedi": "^0.8.0"
   },
   "devDependencies": {
+    "@types/chai": "^4.2.11",
     "@types/mocha": "^7.0.2",
-    "@types/node": "^13.11.1",
+    "@types/node": "^12.12.7",
```

Comment:
Why the downgrade?

Reply:
I want the types to match the node version we are using.